src/redux/saga/employer.ts

```diff
@@ -38,9 +41,29 @@ function* loginEmployerSaga() {
   }
 }
 
+function* logoutEmployerSaga() {
+  try {
+    yield localStorage.removeItem('token')
+  } catch (e) {
+    console.log(e)
+  }
+}
```

Comment:
Why this is also here. I see you already implemented a function to do this inside `Logout` component. So either do it inside saga or in the component itself. I think it's easier to do it inside the component

Reply:
this was my question in the morning's session. I implement it inside the component now.

Reply:
Ok, then let's implement it only inside the `logout` component 👍